pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -319,18 +322,18 @@ func (c *Cluster) assignMons(mons []*monConfig) error {
 		return fmt.Errorf("failed to get available nodes for mons. %+v", err)
 	}
 
-	// if all nodes already have mons return error as we don't place two mons on one node
-	if len(availableNodes) == 0 {
-		return fmt.Errorf("no nodes available for mon placement")
-	}
-
 	nodeIndex := 0
 	for _, m := range mons {
 		if _, ok := c.mapping.Node[m.DaemonName]; ok {
 			logger.Debugf("mon %s already assigned to a node, no need to assign", m.DaemonName)
 			continue
 		}
 
+		// if we need to place a new mon and don't have any more nodes available, we fail to add the mon
```

Comment:
First char in comment should be uppercase.

Reply:
Agreed, the comment casing isn't consistent. Some places lower case is used and other places upper case. I at least try to be consistent with surrounding comments. In this case, surrounding comments are lower case. It seems to reflect developer mood.